src/app/core/data/data.service.ts

```diff
@@ -90,4 +101,15 @@ export abstract class DataService<TNormalized extends CacheableObject, TDomain>
     // return this.rdbService.buildSingle(href));
   }
 
+  findEmbedded<TEmbeddedNormalized extends CacheableObject, TEmbeddedDomain>(
```

Comment:
I have added some additional information about the "search" rels in the rest contract https://github.com/DSpace/Rest7Contract/blob/master/search-rels.md
this idea comes from the Spring Data Rest project, see https://docs.spring.io/spring-data/rest/docs/current/reference/html/#repository-resources.search-resource